### PR TITLE
Adds initial Jukebox docs

### PR DESCRIPTION
**Pull request by captn3m0**
- Refs airsonic/documentation#28
- This doesn't include PulseAudio, which I think deserves mention here, but I haven't managed to set it up yet.

### CODE REVIEW

**Comment by jooola** [comment]
We will get a proper documentation someday , thanks a lot !

@biconou Would you mind taking a quick look at this ? Thanks !

**Comment by biconou** [comment]
Yes I will.

**Comment by captn3m0** [comment]
If someone has any pointers on pulseaudio, I can try it out and document it as well

**Comment by biconou** [comment]
@captn3m0 The document you submitted gave me an idea. 
Wouldn't it be nice to be able, when you configure your jukebox player, to be able to choose and test a special audio device hence there will no need to create a properties file. 
What do you think ? 
You are using the legacy jukebox player aren't you ? Did you try the new one called "java Juke box" ?

**Comment by captn3m0** [comment]
I considered putting the link to the Java Jukebox PR in this doc, but wasn't sure since I didn't see any other open PRs mentioned in the docs. Haven't tried out the branch yet, though.

**Comment by biconou** [comment]
@captn3m0  The java jukebox is already available in Airsonic (what version do you use ?). The current PR is only a fix.

**Comment by captn3m0** [comment]
Forgot to mention: I only tested this on the libresonic docker images, since I didn't have the airsonic one locally and I was testing this offline. I'll try out the airsonic container as well and update the PR if needed.

File: configure/jukebox.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+---
+
+### Jukebox Properties
+
+Jukebox might not always work out-of-the-box and may require some additional tweaking. If you get no sound output while trying to play via the Jukebox, you might need to tweak the audio device being picked up by Java sound. You can run the folowing Java program to get a list of all the audio devices in your system:
+
+<details>
+    <summary>Source Code
+    </summary>
+<div markdown="1">
```
Comment (jooola) [inline configure/jukebox.md]: Why do you have html code here ?
Please remove it

File: configure/jukebox.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        }
+    }
+} 
+```
+</div>
+</details>
+
+<details >
+    <summary>Sample Output</summary>
+<div markdown="1">
```
Comment (jooola) [inline configure/jukebox.md]: Again why html ?

File: configure/jukebox.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+default [default]
+HDMI [plughw:0,3]
+HDMI [plughw:0,7]
+HDMI [plughw:0,8]
+HDMI [plughw:0,9]
+HDMI [plughw:0,10]
+PCH [plughw:1,0]
+```
+</div>
+</details>
```
Comment (jooola) [inline configure/jukebox.md]: Same

File: configure/jukebox.md (hunk 1 above)
Comment (captn3m0) [reply configure/jukebox.md]: It looked better if the code was expandable on demand, just for readability. 

image

(Removed in next commit)

**Comment by captn3m0** [comment]
- Removed HTML
- Tested on the airsonic official docker image

This is what I used to test:

```bash
docker run \
--group-add audio \
--device /dev/snd \
-p 4040:4040 \
-v sound.properties:/usr/lib/jvm/java-1.8-openjdk/jre/lib/sound.properties \
airsonic/airsonic
```

Should be good.

**Comment by captn3m0** [comment]
Also filed https://github.com/airsonic/airsonic.github.io/pull/29 for once this is merged (has nav changes)

**Comment by jooola** [comment]
Thanks ! I just wait for @biconou 's small review and then we can merge it !

**Comment by biconou** [comment]
@jooola sorry I was too busy at work.

**Comment by jooola** [comment]
@biconou No problem ! Next time ^^

**Pull request merged**